ansible/site.yml

```diff
@@ -194,6 +201,8 @@
           parameters:
             key: "pvwproxy.proxy_file_path"
             value: "/opt/hpccloud/proxy"
+      when: not cumulus
```

Comment:
Can these conditioned ```when: not cumulus``` tasks be placed into a separate ```hpccloud.yml``` and conditionally included?